app/forms/ChangeYourRoleImporterFormProvider.scala

```diff
@@ -17,10 +17,11 @@
 package forms
 
 import forms.mappings.Mappings
-import models.requests.DataRequest
 import play.api.data.Form
 
-class ChangeYourRoleImporterForm extends Mappings {
+import javax.inject.Inject
+
+class ChangeYourRoleImporterFormProvider @Inject() extends Mappings {
```

Comment:
Please remove injection so should just be `class ChangeYourRoleImporterFormProvider extends Mappings {`. Do this for all form providers apart from `UploadAnotherSupportingDocumentFormProvider`, thanks.